CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
+
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language, and we would like you to join us in improving Gonum's quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+Questions about Gonum or the use of its libraries can be directed to [gonum-dev](https://groups.google.com/forum/#!forum/gonum-dev).
+
+#### Table of Contents
+
+[Project Scope](#project-scope)
+
+[The Gonum Repositories](#the-gonum-repositories)
+
+[Contributing](#Contributing)
+  * [Working Together](#working-together)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Code Contribution](#code-contribution)
+  * [Code Review](#code-review)
+  * [What Can I Do to Help?](#what-can-i-do-to-help)
+  * [Style](#style)
+
+## Project Scope
+
+The purpose of the Gonum project is to provide general purpose numerical computing and data science libraries for the Go programming language.
+The libraries should aim to provide building blocks for disciplinary work and advanced algorithms.
+Code should be implemented in pure Go.
```

Comment:
From here on the paragraph looks like it belongs in Contributing section rather than Project Scope.